spec/hashie/extensions/indifferent_access_with_rails_hwia_spec.rb

```diff
@@ -35,6 +36,10 @@ class CoercableHash < Hash
     include Hashie::Extensions::MergeInitializer
   end
 
+  class MashWithIndifferentAccess < Hashie::Mash
```

Comment:
@dblock  The text at the top of `indifferent_access_with_rais_hwia_spec` says: 
```
# This set of tests verifies that Hashie::Extensions::IndifferentAccess works with
# ActiveSupport HashWithIndifferentAccess hashes. See #164 and #166 for details.
```

Therefor I think the place of the test is in this file and not in a separate one; However, if you insist I'll move it to a separate file - and change the comment at the top of `indifferent_access_with_rais_hwia_spec`